src/lib.rs

```diff
@@ -120,6 +120,14 @@ impl From<[u8; 64]> for fp_480::Fp480 {
     }
 }
 
+impl From<[u8; 64]> for fp_480::Monty {
```

Comment:
shouldn't this be 60 instead of 64?

Reply:
We have a 60, but the 64 is for when we do 2 sha256 values concatenated together.